ScnSoft-Education/LeetCodeTraining/SortColors/SortColorsSolution.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace LeetCodeTraining.SortColors
+{
+    //https://leetcode.com/problems/sort-colors/
+    public static class SortColorsSolution
+    {
+        public static void SortColors(int[] nums)
+        {
+            int lp = 0;
```

Comment:
hello @shomzascnsoft 
initially they meant Red pointer and Blue pointer, as pointers to indeces where Red color (0's) end, and Blue color (2's) start. Initially I had wptr as well (White pointer) but soon I got rid of it because we need only two pointers to split array to three parts. I renamed it to lp and rp as _left pointer_ and _right pointer_  to get rid of this discrepancy where one index means end while the other index means start, now lp and rp are just "splitters" of array.